### DISCUSSION TITLE
Elsa 2.0 Getting Started

### DISCUSSION THREAD

**Discussion by hpposch** [elsa-workflows/elsa-core#699] [Q&A]
Hi all,

I would like to develop a new application based on Elsa 2.0.
But I can't get the examples running.
I try to start the project 'ElsaDashboard.Samples.Monolith'
And then I navigate to the URL that is defined inside the 'launchSettings.json'
But I do not see the Dashboard.
What is wrong?

I also wanted to know if there is any Api description available, so that one can develop new Client application for Elsa.

Thanks in advance for your help! :)

**Comment by hpposch** [comment]
I used the branch /feature/elsa-2.0
Maybe this is the problem?!

**Comment by hpposch** [comment]
I got the Example running now and i see the Sample Workflows but when i click on edit, nothing happens.
Also when i want to create a new Workflow there are no Activities inside the menu on the right side..

**Comment by sfmskywalker** [comment]
Hi!

So the dashboard + designer are currently under heavy development, so there are bound to be issues and lack of functionality (such as "Edit" not doing anything).

Things should start to look better in the following weeks.

> I also wanted to know if there is any Api description available, so that one can develop new Client application for Elsa.

Right now there's Swagger documentation that will be generated dynamically. If you launch the Monolith sample application, you should be able to navigate to: https://localhost:3956/swagger

Thanks for getting started!

**Comment by hpposch** [comment]
Alright,
thanks for your answer.
I just got the swagger Documentation.

Since I am a WPF Developer i would try to create a WPF client and a library that others could use later on.

When do you plan to release Version 2 of Elsa?

**Comment by sfmskywalker** [comment]
That would be super awesome.
I hope to release V2 around end of March.

**Comment by dailyUnknown** [comment]
@sfmskywalker, do you have or know a website providing paid technical support for ELSA? If I decide to use Version 2, I think I can sign up for a paid subscription of technical support.

Thank you.

**Reply by sfmskywalker** [reply]
There's currently no such thing. The closest thing to paid support is reaching out to one of our contributors directly and have them invoice you for time spent. That being said, I do realize it would be helpful to be able to pay for support when you need it, so it's something to consider.

**Reply by dailyUnknown** [reply]
Thank you sfm.

**Comment by craigfowler** [comment]
@dailyUnknown I certainly don't think that such a thing exists (at the moment, anyway).  If it does then @sfmskywalker would be the one to provide it himself (!).

Is there anything else you need from this issue, or is it safe to close (or perhaps move to a discussion)?  Thanks.

**Comment by dailyUnknown** [comment]
Move to a discussion sounds perfect. Thank you.
